README.md

```diff
@@ -5,6 +5,8 @@
 
 The project TBD [Tau Be Damned] aims to use the amino acid sequence of a protein to identify whether it is disordered.
 
+![The TBD logo, nice isn't it?](https://github.com/Intrinsically-Disordered/TBD/blob/main/TBDlogo.png?raw=True)
```

Comment:
It's actually not quite correct, as you can see from here: https://github.com/Intrinsically-Disordered/TBD/tree/updates#project-tbd-tau-be-damned

Reply:
No worries. I'll modify this and the following images